### ISSUE TITLE
Cannot load assembly on UWP

### ISSUE THREAD

**Issue by TheAlmightyBob** [TheAlmightyBob/Calendars#50]
Trying to use the latest version (0.6.45-beta) in a UWP project fails with:
>Could not load file or assembly 'Plugin.Calendars, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

**Comment by jphellemons** [comment]
I am not sure, but I think that I might have found it.
https://github.com/TheAlmightyBob/Calendars/blob/master/Calendars.nuspec#L59
I believe that line 59 and 60 should be:

````
<file src="Calendars\Calendars.Plugin.UWP\bin\Release\Plugin.Calendars.pdb" target="lib\UAP10\Plugin.Calendars.pdb" />
<file src="Calendars\Calendars.Plugin.UWP\bin\Release\Plugin.Calendars.xml" target="lib\UAP10\Plugin.Calendars.xml" />
````

check the file extensions. The **.xml** and **.pdxmlb**

**Comment by TheAlmightyBob** [comment]
@jphellemons I saw that and tried fixing it already, but it didn't make a difference for this issue. If it solves it for you, please let me know.

**Comment by jphellemons** [comment]
Well the error looks like there was/is something wrong with the missing dll's which are mentioned in the nuspec. I often build uwp as x86. But the nuget is (of course) 'any cpu' so the path seems valid. 
Will try to test it with this guide https://blogs.endjin.com/2014/07/how-to-test-nuget-packages-locally/

**Comment by jphellemons** [comment]
I have looked at the Xamarin Forms nuspec template 
https://github.com/jamesmontemagno/Xamarin-Templates/blob/master/Templates/NuSpecLibrary/Plugin.nuspec#L98
and there is on line 101 a reference to a xml file _Properties\Feature.Plugin.Uwp.rd.xml_
So I have set this file to embedded resouce so that we could add this reference to the nuspec file too.
However, being an almost empty file. I don't think that this will do much?

<img width="146" alt="embedded-resource" src="https://user-images.githubusercontent.com/1119134/39617229-23054822-4f7f-11e8-9ba1-481eec878097.PNG">

**Comment by jphellemons** [comment]
Ow and both the uwp and the uwp test project can update the package 'Microsoft.NETCore.UniversalWindowsPlatform' from 6.0.6 to 6.1.2 which I did not commit to my PR but I updated them locally. Perhaps re-open this issue, since it's not confirmed fixed (yet)?

**Comment by TheAlmightyBob** [comment]
Maybe you've already figured out NuGet, but you can grab the package from the AppVeyor build of your merge request:
https://ci.appveyor.com/project/TheAlmightyBob/calendars/build/0.6.50-beta-ocenshje/artifacts

(also I didn't intend to close this... I guess GitHub did that automatically because your PR referenced it...)

**Comment by jphellemons** [comment]
Thank you, I did not figure it out. Will try this version. But this version does not contain the missing xml which might cause the whole missing dll for uwp thing. Because of the conflict in my latest commit for the pr. https://github.com/TheAlmightyBob/Calendars/pull/52/conflicts

**Comment by jphellemons** [comment]
I have downloaded ILSpy 3.1 https://github.com/icsharpcode/ILSpy/releases/tag/v3.1-final and drag and dropped the .nupkg in it. (found [here ](https://github.com/icsharpcode/ILSpy/issues/986) that it was possible) And also tried https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/releases
<img width="368" alt="ilspy" src="https://user-images.githubusercontent.com/1119134/39693033-f1619cc4-51e2-11e8-896b-cdcd20b75ed8.PNG">
Will make a new commit to add some runtimes in the nuspec as it says in the nuget docs. https://docs.microsoft.com/en-us/nuget/guides/create-uwp-packages

**Comment by TheAlmightyBob** [comment]
@jphellemons This specific issue should now be fixed in the latest build, but it's not published to nuget because it just revealed another issue with the UWP library itself.

I'm now seeing this error when trying to call the library:
```
Could not load file or assembly 'System.Runtime.WindowsRuntime, Version=4.0.11.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
```

Not yet sure what the problem is. Currently looking at the `project.json` since that's the main thing that changed as part of the .netstandard update (the error doesn't happen in the previous version).

**Comment by TheAlmightyBob** [comment]
I _think_ that the error I was seeing was actually just an incompatibility of the CalendarsTester project, so optimistic now that this is fixed and I just need to update that project.

**Comment by TheAlmightyBob** [comment]
Fixed via #56 and released in 0.6.64-beta

**Comment by jphellemons** [comment]
Thank you Bob! I have seen you using the 1.0.0.0 in the assemblyinfo. I thought that AppVeyor required the {version}. But it's so cool that it is fixed 👍 :shipit:
